<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommodoreHomeForm</class>
 <widget class="QWidget" name="CommodoreHomeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_alarm">
     <property name="text">
      <string>Alarm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_alarm">
     <property name="text">
      <string>Activated</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_music">
     <property name="text">
      <string>Music</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_music">
     <property name="maxVisibleItems">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Dimmer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QSlider" name="horizontalSlider_dimmer">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>25</number>
       </property>
       <property name="pageStep">
        <number>25</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_dimmer">
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
